### ISSUE TITLE
Screenshots direct in bamboo

### ISSUE THREAD

**Issue by foal** [saucelabs/bamboo_sauce#23]
It is possible to have direct links to screenshot/video or even show them as image collection directly in bamboo?

**Comment by rossrowe** [comment]
Yes, this should be possible, as I think the Sauce REST API will provide links to the screenshots/videos, I'll see how I can integrate these into the plugin UI

**Comment by halkeye** [comment]
@foal We now have links to the video and logs in the sauce results tab in 1.6.63
Does this fix the problem?

cc @jsmoxon

**Comment by foal** [comment]
Will try new version and let you know.

**Comment by foal** [comment]
No, It doesn't fix the problem. New functionality is Ok with video, but doesn't provide the access to screenshots. 

It will be perfect if plugin will present the screenshots not as the links but as `img` elements
